dmgroups.tex

```diff
@@ -0,0 +1,105 @@
+The following mandate established the DMLT:
+
+\begin{itemize}
+\item Charter/purpose
+	\begin{itemize}
+	\item Maintain scope of work and keep within resource allocation across DM
+	\item Ensure overall project management integrity across DM
+	\item Ensure Earned Value management requirements are met
+	\end{itemize}
+\item Membership
+	\begin{itemize}
+	\item Co-Chaired by the DM Project Manager  and  DM Project Scientist
+	\item Lead Institution Technical/Control Account Managers (T/CAMs or CAMs)
+	\item Institutional Scientists
+	\item Members of the DM Systems Engineering Team (\secref{sect:sysengt})
+	\end{itemize}
+\item Responsibilities
+	\begin{itemize}
+	\item Prepares all budgets, schedules, plans
+	\item Meets every week to track progress, address issues/risks, adjust work assignments and schedules, and disseminate/discuss general PM communications
+	\item Creates and publishes monthly, quarterly, annual progress reports
+	\item Meets at start of each software development phase with SAT to establish detailed scope/work plan
+	\item Meets with SAT for change control (DMCCB)
+	\end{itemize}
+\end{itemize}
+
+The DM Leadership Team and the DM System Engineering Team (\secref{sect:sysengt}) work in synchrony.
+The DMLT makes sure the requirements and architecture/design are estimated and scheduled in accordance with LSST Project required budgets and schedules.
+
+ \subsubsection{Communications}
+ A mailing list\footnote{\url{lsst-dmlt@listserv.lsstcorp.org}} exists for DMLT related messages.
+ On Mondays the DMLT hold a brief telecon(30 to 45 minutes) , this telecon serves to :
+\begin{itemize}
+\item Allow the Project manager and DM Scientist  to pass on important project level information and general guidance.
+\item Raise any blocking or not well understood issues across DM - this may result in calling a splinter meeting to further discuss with relevant parties.
+\item Inform everyone of any LCRs in process at LSST level which may be of interest to or  have impact on DM
+\item Check on outstanding actions on DMLT members.
+\end{itemize}
+
+Face to Face meetings of DM are held two times a year these are opportunities to:
+\begin{itemize}
+\item Discuss detailed planning for the next cycle
+\item Discuss technical topics in a face to face environment
+\item Work together on critical issues
+\item Help make DM function as a team
+\end{itemize}
+
+The T/CAMs of each of the institutions meet via video on Tuesdays and Fridaysfor a short "standup" meeting to ensure that any cross-team issues are surfaced and resolved expeditiously.
+This meeting is chaired by the Deputy Project Manager.
+Each T/CAM notes any significant progress of interest to other teams and any problems or potential problems that may arise.
+
+
+\subsection{DM Change Control Board \label{sect:dmccb}}
+The DMCCB has responsibility for issues similar to those of the LSST Change Control Board, but restricted to those contained within the DM subsystem.
```

Comment:
Currently the TCT's job is to recommend changes be approved by the DM Project Manager. The TCT does not do the approval (this is similar to the project-wide CCB where the CCB recommend acceptance but Victor still has to sign off). This paragraph says that the DMCCB approves things (the JIRA RFC workflow calls out the recommendation explicitly)